auditlog_tests/tests.py

```diff
@@ -1104,6 +1107,13 @@ def test_register_from_settings_register_models(self):
         self.assertEqual(fields["include_fields"], ["label"])
         self.assertEqual(fields["exclude_fields"], ["text"])
 
+    def test_registration_error_if_bad_serialize_params(self):
+        with self.assertRaises(AuditLogRegistrationError):
```

Comment:
Please test the error message here as well. like https://github.com/jazzband/django-auditlog/blob/a24b79af0c558c50381e752377e3003f0c53b564/auditlog_tests/tests.py#L1027-L1030

Reply:
Good call. fixed.